cms/djangoapps/contentstore/tasks.py

```diff
@@ -668,7 +669,19 @@ def read_chunk():
 
 @shared_task
 @set_code_owner_attribute
-def update_outline_from_modulestore_task(course_key_str):
+def update_multiple_outlines_from_modulestore_task(course_key_list: List[str]):
+    """
+    Celery task that creates multiple celery tasks - one per learning_sequence course outline
+    in the passed-in list.
+    """
+    for course_key_str in course_key_list:
+        # Perform all the error-checking in the child celery tasks.
+        update_outline_from_modulestore_task.delay(course_key_str)
```

Comment:
I didn't do any key-checking here since it's done in the single-course task. Thoughts?

Reply:
I do the checking before task invocation in the management command as a minor optimization, but I don't think it really matters that much.

Reply:
I agree - less Celery tasks (upon invalid key) beats checking each key twice. Will change.